### DISCUSSION TITLE
Not standart nozzle diameter.

### DISCUSSION THREAD

**Discussion by kod1411** [OrcaSlicer/OrcaSlicer#6395] [Ideas]
I have not found in either BambuStudio or OrcaSlicer the possibility to set the nozzle diameter to 0.3mm or 0.5mm for X1C, and it would be great to have this possibility.
Does anyone know anything about this feature?

**Comment by foreachthing** [comment]
My guess is, you'd have to create your own profile for these nozzle diameters:

![image](https://github.com/user-attachments/assets/b0c0769b-90f5-407b-82f6-7588f5372643)

It's not difficult at all. Just edit this value (see image above) and save the profile under a new name. Done.

**Reply by mrelwood** [reply]
The latest versions of Orca Slicer will not print if the nozzle diameter set in the Printer Parts in the Device tab isn't consistent with the Printer settings you mentioned. But the Printer Parts doesn't have a 0.3mm option, and I don't think it can be modified. Because of this I can no longer use a 0.3mm nozzle. If someone knows a solution, please let us know.
